### PR TITLE
hotfix: improve docs

### PR DESCRIPTION
**Pull request by JakeSCahill**
# Description

Add docs to .github directory to keep the repo tidy.

Updates the templates to use the latest GitHub features.

## Type of change

- Documentation fix

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# Contribute to Trinity
```
Comment (rajivshah3) [inline .github/CONTRIBUTING.md:1]: Should we just move `CONTRIBUTING.md` to `.github/CONTRIBUTING.md`?

**Review by cvarley100 — APPROVED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (cvarley100) [reply .github/CONTRIBUTING.md:1]: How do you mean?

Comment (rajivshah3) [reply .github/CONTRIBUTING.md:1]: There's already a similar doc called `CONTRIBUTING.md`:  https://github.com/iotaledger/trinity-wallet/blob/develop/CONTRIBUTING.md

**Comment by rajivshah3** [comment]
Superseded by #2672

**Pull request closed without merging**